handle custom docker for none xsoar supported packs (#3156)
demisto/demisto-sdk: CHANGELOG.md

* Removed the `timezone_offset` argument in the *modeling-rules test* command.
* Fixed an issue where **lint** failed when importing functions from CommonServerUserPython.
* The **format** command now will sync hidden parameters with master branch.

> * **validate** will no longer throw errors for custom dockers in case of none xsoar supported pack.

Reviewer: ```suggestion
* **validate** will only fail on docker related errors if the pack is supported by xsoar.
```
